src/pages/graphql/usage/caching.md

```diff
@@ -77,6 +77,11 @@ To define additional factors for computing `X-Magento-Cache-Id` hash values, add
 </type>
 ```
 
+<InlineAlert variant="warning" slots="text"/>
+
+X-Magento-Cache-Id response header is only compatible with Varnish or Fastly.  It cannot be reliably used with built-in Full Page Cache.
```

Comment:
```suggestion
The `X-Magento-Cache-Id` response header is only compatible with Varnish or Fastly.  It cannot be reliably used with built-in Full Page Cache.
```

Also, you have two blank lines beneath this warning. Please delete one of them.

Reply:
Right, wasn't sure how the inline alerts worked.  Fixed.